jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -589,6 +589,7 @@ fi
 
 IGN_PHYSICS_DEPENDENCIES="libbenchmark-dev \\
                           dart6-data \\
+                          libdart6-collision-bullet-dev \\
```

Comment:
do we want to make it conditional to a given version (or above) of ign-physics? In that case, that logic starts in line 601.

Reply:
Ok, check out 9a03d4c